content/service-manuals.md

```diff
@@ -20,22 +20,26 @@ redirect_from:
+| Gazelle      |     | [gaze16](https://tech-docs.system76.com/models/gaze16/repairs.html)
 | Lemur        | [lemu7](https://github.com/system76/docs/blob/gh-pages/service-manuals/pdfs/Lemur/lemu7-service-manual.pdf), [lemp9](https://github.com/system76/docs/blob/gh-pages/service-manuals/pdfs/Lemur/lemp9-service-manual.pdf) | [lemp9](https://tech-docs.system76.com/models/lemp9/repairs.html), [lemp10](https://tech-docs.system76.com/models/lemp10/repairs.html)
 | Oryx Pro     | [oryp5](https://github.com/system76/docs/blob/gh-pages/service-manuals/pdfs/Oryx/oryp5-service-manual.pdf)  | [oryp6](https://tech-docs.system76.com/models/oryp6/repairs.html), [oryp7](https://tech-docs.system76.com/models/oryp7/repairs.html)
 | Pangolin     | | [pang10](https://tech-docs.system76.com/models/pang10/repairs.html)
 | Serval WS    | [serw11](https://github.com/system76/docs/blob/gh-pages/service-manuals/pdfs/Serval/serw11-service-manual.pdf) | [serw12](https://tech-docs.system76.com/models/serw12/repairs.html)
 
 | Desktops       | PDF | Tech Docs |
-| ---------------| ----| ----------|
-| Meerkat        |   | [meer5](https://tech-docs.system76.com/models/meer5/repairs.html)
-| Thelio         | [thelio-r1](https://github.com/system76/docs/blob/gh-pages/service-manuals/pdfs/Thelio/R1/thelio-r1-service-manual.pdf), [thelio-b1](https://github.com/system76/docs/blob/gh-pages/service-manuals/pdfs/Thelio/B1/thelio-b1-service-manual.pdf) |
-| Thelio Major   |   | [thelio-major](https://tech-docs.system76.com/models/thelio-major-intel-and-amd/repairs.html)
-| Thelio Massive | | [thelio-massive](https://tech-docs.system76.com/models/thelio-massive-b1.2/repairs.html)
-| Thelio Mega    |   | [thelio-mega](https://tech-docs.system76.com/models/thelio-mega-r1.0/repairs.html)
-| Thelio Mira    | | [thelio-mira](https://tech-docs.system76.com/models/thelio-mira-r1.0/repairs.html)
+|:--------------:|:---:|:---------:|
+| Meerkat        |     | [meer5](https://tech-docs.system76.com/models/meer5/repairs.html)
+| Thelio         |     | [thelio-r1](https://github.com/system76/docs/blob/gh-pages/service-manuals/pdfs/Thelio/R1/thelio-r1-service-manual.pdf), [thelio-b1](https://github.com/system76/docs/blob/gh-pages/service-manuals/pdfs/Thelio/B1/thelio-b1-service-manual.pdf) |
+| Thelio Major   |     | [thelio-major](https://tech-docs.system76.com/models/thelio-major-intel-and-amd/repairs.html)
+| Thelio Massive |     | [thelio-massive](https://tech-docs.system76.com/models/thelio-massive-b1.2/repairs.html)
+| Thelio Mega    |     | [thelio-mega](https://tech-docs.system76.com/models/thelio-mega-r1.0/repairs.html)
+| Thelio Mira    |     | [thelio-mira](https://tech-docs.system76.com/models/thelio-mira-r1.0/repairs.html)
+
+| Accessories | PDF | Tech Docs |
+|:-----------:|:---:|:---------:|
+| Launch      |     | [launch_1](https://tech-docs.system76.com/models/launch_1/README.html)
```

Comment:
Looks like all the other tech-docs links link to the repairs page for each model. Any reason not to do the same with Launch? https://tech-docs.system76.com/models/launch_1/repairs.html

Reply:
It may be time to re-evaluate that decision now that guides have been removed (and replaced with external overview.) Just a thought.

Reply:
Fair point. I'll let Support decide what they want to do, but I'd still advise consistency across all links to tech-docs.